tree/tree/Extension/Extension+UIView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Extension+UIView.swift
+//  tree
+//
+//  Created by hyeri kim on 30/01/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    func applyShadow(shadowView: UIView, width: CGFloat, height: CGFloat) {
```

Comment:
[로직]
UIView's Extension에 해당 함수를 선언했다면, shadowView 파라미터를 만들어서 하위 뷰의 Layer를 불러오지 않고 바로 변경하고자 하는 View에 해당 함수를 적용하는게 더 간결하지 않을까 생각됩니다.

```suggestion
    func applyShadow(width: CGFloat, height: CGFloat) {
    let shadowPath = UIBezierPath(roundedRect: self.bounds, cornerRadius: 14.0)
    
    // 호출: self.shadowView.applyShadow(width: 0, height: 0)
```